internal/test/net.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I'm not sure we should add an entire package just for this method. I dont think there are other places in the codebase that have subpackages just for test utils. I'd suggest either copy paste the freeport code whereever its needed or simply add phayes/freeport as a new dependency

Reply:
I only added this because the `internal/test` package already exists – it currently contains the `HelmHome` and `TempDir` functions, so I don't see a reason not  to add this to the package too.

Reply:
i see it there now, ok - plz ignore